discovery-provider/plugins/notifications/package.json

```diff
@@ -9,7 +9,7 @@
     "lint:fix": "npm run lint -- --fix",
     "pretty": "prettier --write .",
     "sql-ts": "ts-node scripts/sql-ts.ts",
-    "start": "pm2-runtime build/src/main.js --restart-delay=3000",
+    "start": "pm2-runtime build/src/main.js --restart-delay=3000 --node-args=\"--max-old-space-size=512\"",
```

Comment:
probably don't need this when using the correct npm command but it doesn't hurt for now